libbeat/docs/release-notes/5.0.0.asciidoc

```diff
@@ -0,0 +1,240 @@
+- Fix dynamic configuration int/uint` to float type conversion. {pull}2698[2698]
+- Fix primitive types conversion if values are read from environment variables. {pull}2698[2698]
+
+
+*Filebeat*
+
+- Stop filebeat if started without any prospectors defined or empty prospectors {pull}644[644] {pull}647[647]
+- Improve shutdown of crawler and prospector to wait for clean completion {pull}720[720]
+- Omit `fields` from Filebeat events when null {issue}899[899]
+- Improvements in registrar dealing with file rotation. {pull}1281[1281]
+- Multiline reader normalizing newline to use `\n`. {pull}1552[1552]
+- Fix potential data loss between Filebeat restarts, reporting unpublished lines as published. {issue}2041[2041]
+- Fix open file handler issue. {issue}2028[2028] {pull}2020[2020]
+- Fix async publisher sending empty events {pull}2455[2455]
+- Fix potential issue with multiple harvester per file on large file numbers or slow output {pull}2541[2541]
+- Fix input buffer on encoding problem. {pull}2416[2416]
+- Fix issue when `clean_removed` and `clean_inactive` were used together that states were not directly removed from the registry.
+- Fix issue where upgrading a 1.x registry file resulted in duplicate state entries. {pull}2792[2792]
+
+
+*Packetbeat*
+
+- Create a proper BPF filter when ICMP is the only enabled protocol {issue}757[757]
+- Check column length in pgsql parser. {issue}565[565]
+- Harden pgsql parser. {issue}565[565]
+- Add missing nil-check to memcached GapInStream handler. {issue}1162[1162]
+- Fix NFSv4 Operation returning the first found first-class operation available in compound requests. {pull}1821[1821]
+- Fix TCP overlapping segments not being handled correctly. {pull}1898[1898]
+- Fix mapping for some Packetbeat flow metrics that were not marked as being longs. {issue}2177[2177]
+- Fix handling of messages larger than the maximum message size (10MB). {pull}2470[2470]
+- Fix the `bpf_filter` setting. {issue}2660[2660]
+- Fix compile issues for OpenBSD. {pull}1347[1347]
+
+
+*Winlogbeat*
+
+- Fix panic when reading messages larger than 32K characters on Windows XP and 2003. {pull}1498[1498]
+- Fix panic that occurs when reading a large events on Windows Vista and newer. {pull}1499[1499]
+- Adding missing argument to the "Stop processing" log message. {pull}1590[1590]
+- Fix issue with rendering forwarded event log records. {pull}1891[1891]
+- Fix potential data loss between Winlogbeat restarts, reporting unpublished lines as published. {issue}2041[2041]
+- Fix corrupt registry file that occurs on power loss by disabling file write caching. {issue}2313[2313]
+
+
+
+==== Added
+
+*Affecting all Beats*
+
+- Update to Go 1.7. {pull}2306[2306]
+- Add option to Elasticsearch output to pass http parameters in index operations {issue}805[805]
+- Improve Logstash and Elasticsearch backoff behavior. {pull}927[927]
+- Add experimental Kafka output. {pull}942[942] {pull}2188[2188] {pull}2190[2190] {pull}2284[2284]
```

Comment:
@urso Is the Kafka output experimental at this point? We don't seem to mark it as such in the docs. I'd go with fully supported.